code/index.html

```diff
@@ -1,22 +1,114 @@
+
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie-edge">
+    <title>The Daily Deep Dive</title>
+    <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
+  <body class="body">
     <header>
-      <!-- Some kind of logo and navbar -->
+    
+      <img src="assets/jellyfish.png" alt="logo" class="logo">
+      <h1>THE DAILY DEEP DIVE</h1>
+      <nav>
+        <a href="#">DEEP</a>
+        <a href="#">DEEPER</a>
+        <a href="#">DEEPEST</a>
+      </nav>
+
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      
     </header>
+    <section class="big-news-container">
+      <img src="assets/danist-bviex5lwf3s-unsplash.jpg" alt="Jelly Fish">
+      <div class="news-text">
+        <h2 class="h2">Take a Deep Breath</h2>
+        <p class="heading-two"><strong>One, two, three...</strong></p></p>
+        <p class="hedline-two"><strong>Lets dive down to where there are no up so or down!</strong></p>
```

Comment:
Do these two need different class names? Also, try to keep the styling to the CSS file (`font-weight`)